distributions/distribution-resources/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -28,7 +28,7 @@ log4j.appender.out.maxBackupIndex=10
 # File appender - events.log
 log4j.appender.event=org.apache.log4j.RollingFileAppender
 log4j.appender.event.layout=org.apache.log4j.PatternLayout
-log4j.appender.event.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss.SSS} [%-20.20c{1}] - %m%n
+log4j.appender.event.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss.SSS} [%c{1}] - %m%n
```

Comment:
As already stated, it is personal taste. I like FQNs regardless if it is always "smarthome.events.something". So a UI could handle all log files the same way without knowing that this log file is using that package prefix. I never stated that you have to change it in that configuration.